src/javascript/app/pages/trade/purchase.js

```diff
@@ -63,15 +63,20 @@ const Purchase = (() => {
             container.style.display = 'block';
             message_container.hide();
             confirmation_error.show();
-            let message = error.message;
-            if (/RestrictedCountry/.test(error.code)) {
+
+            let message;
+            if (/NoMFProfessionalClient/.test(error.code)) {
+                message = localize('[_1][_2]In the EU, financial binary options are only available to professional investors.[_3][_4][_5]Apply now as a professional investor[_6][_7][_8]', ['<div class="gr-row font-style-normal">', '<div class="gr-12 gr-padding-20">', '</div>', '<div class="gr-12 gr-padding-20">', `<a id="btn_professional" class="button" href="${urlFor('user/settings/professional')}"><span>`, '</span></a>', '</div>', '</div>']);
```

Comment:
I'm confused, why not just combine placeholders together? e.g. `[_1][_2]` can become `[_1]` and replaced by `'<div class="gr-row font-style-normal"><div class="gr-12 gr-padding-20">'` etc so translators' job becomes easier as well.

We can even separate them to be completely without placeholders, i.e. make the divs using jquery elements, then set the inner text to be the localized sentence 1 and then localized sentence 2.